src/Equinor.ProCoSys.BusReceiver/PcsSubscriptionClient.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.BusReceiver.Interfaces;
+using Microsoft.Azure.ServiceBus;
+
+namespace Equinor.ProCoSys.BusReceiver
+{
+    public class PcsSubscriptionClient : SubscriptionClient, IPcsSubscriptionClient
+    {
+        public PcsTopic PcsTopic { get; }
+
+        private Func<IPcsSubscriptionClient, Message, CancellationToken, Task> _pcsHandler;
+        public PcsSubscriptionClient(string connectionString, PcsTopic pcsTopic, string subscriptionName)
+            : base(connectionString, pcsTopic.ToString(), subscriptionName, ReceiveMode.PeekLock, RetryPolicy.Default) =>
+            PcsTopic = pcsTopic;
+
+        public void RegisterPcsMessageHandler(Func<IPcsSubscriptionClient, Message, CancellationToken, Task> handler, MessageHandlerOptions messageHandlerOptions)
+        {
+            _pcsHandler = handler;
+            base.RegisterMessageHandler(HandleMessage, messageHandlerOptions);

```

Comment:
remove base.

Reply:
I find code that uses their full qualifier `base.<...>` simpler to read, instead of guessing if this is a local or inherited property/method.